grub.d/99_migration

```diff
@@ -14,29 +14,14 @@ else
     ) ${CLASS}"
 fi
 
-migration_iso=$(echo /migration-image/*-Migration.*.iso)
+migration_iso=$(echo /*-Migration.*.iso)
```

Comment:
This change is not correct and should be reverted. The installation target is in `/migration-image/...`
See the SLES15-Migration package from home:marcus.schaefer:dms (or your branch)

```
rpm -qpl SLES15-Migration-2.0.40-1.x86_64.rpm
/migration-image
/migration-image/SLES15-Migration.x86_64-2.0.40-Build1.9.iso
/usr/sbin/run_migration
```

Thanks